src/librustc_codegen_ssa/traits/misc.rs

```diff
@@ -11,14 +11,14 @@ pub trait MiscMethods<'tcx>: BackendTypes {
         &self,
     ) -> &RefCell<FxHashMap<(Ty<'tcx>, Option<ty::PolyExistentialTraitRef<'tcx>>), Self::Value>>;
     fn check_overflow(&self) -> bool;
-    fn instances(&self) -> &RefCell<FxHashMap<Instance<'tcx>, Self::Value>>;
+    fn instances(&self) -> &RefCell<FxHashMap<Instance<'tcx>, Self::FuncId>>;
```

Comment:
I don't think this is correct, isn't this used for both statics and functions?

Reply:
You are right:

https://github.com/rust-lang/rust/blob/152527f443c51517bb867befa93809ce5b9b1cd1/src/librustc_codegen_llvm/mono_item.rs#L34